Lib/calendar.py

```diff
@@ -180,6 +186,32 @@ def itermonthdays2(self, year, month):
         for i, d in enumerate(self.itermonthdays(year, month), self.firstweekday):
             yield d, i % 7
 
+    def itermonthdays3(self, year, month):
```

Comment:
The order of methods: itermonthdays2, itermonthdays3, itermonthdays4, itermonthdays looks slightly inconsistent.